metricflow/model/validations/unique_valid_name.py

```diff
@@ -44,6 +45,8 @@ def get_reserved_reason(keyword: MetricFlowReservedKeywords) -> str:
                 "Used as the query input for creating time series metrics from measures with "
                 "different time dimension names."
             )
+        if keyword is MetricFlowReservedKeywords.MF_INTERNAL_UUID:
```

Comment:
I'm pretty sure this has to be an `elif`

Reply:
doesn't need to, but it'll be structurally more correct I think. So changing it